project/src/main/world/career-world.gd

```diff
@@ -60,14 +104,18 @@ func _move_camera(percent: float) -> void:
 ## Returns:
 ## 	A dictionary defining 'min_value' and 'max_value' properties corresponding to the leftmost/rightmost x position
 ## 	within the _player_path2d
```

Comment:
Change this comment to be more descriptive and accurate; the range is narrower than the path2d's range

Reply:
Expanded comment: `the range of camera x values is slightly narrower ...`